apps/docs/content/guides/auth/social-login/auth-kakao.mdx

```diff
@@ -132,6 +132,51 @@ suspend fun signOut() {
 </TabPanel>
 </Tabs>
 
+## Using Kakao Login JS SDK
+
+[Kakao Login JS SDK](https://developers.kakao.com/docs/latest/en/kakaologin/js) is an official Kakao SDK for authenticating Kakao users on websites.
+
+You need an code to get [ID Token with the issued authorization code](https://developers.kakao.com/docs/latest/en/kakaologin/common#login-with-oidc).
+
+For example, this code shows a how to get ID Token:
+
+```
+const requestUrl = new URL(request.url);
+const code = requestUrl.searchParams.get('code');
+
+if (code) {
+  const res = await fetch('https://kauth.kakao.com/oauth/token', {
```

Comment:
Might be missing something but is there a reason for using  a POST request instead of [Sign in with OAuth](https://supabase.com/docs/reference/javascript/auth-signinwithoauth) ?